openedx/adg/lms/student/helpers.py

```diff
@@ -0,0 +1,163 @@
+"""
+Helpers for edx student app
+"""
+from django.conf import settings
+from django.contrib.auth.tokens import default_token_generator
+from django.urls import reverse
+from django.utils.http import int_to_base36
+
+from openedx.adg.common.lib.mandrill_client.client import MandrillClient
+from openedx.adg.common.lib.mandrill_client.email_data import EmailData
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangolib.markup import Text
+from student.models import UserProfile
+
+
+def compose_and_send_activation_email(user, activation_key):
+    """
+    Prepare and send email for account activation
+
+    Arguments:
+        user (User): Django user object
+        activation_key (str): Activation key that will be send as email content
+
+    Returns:
+        None
+    """
+    user_profile = UserProfile.objects.get(user=user)
+    root_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
+    activation_url = '{root_url}/activate/{activation_key}'.format(
+        root_url=root_url,
+        activation_key=activation_key
+    )
+
+    context = {
+        'first_name': user_profile.name.split()[0],
+        'activation_link': activation_url
+
+    }
+
+    email_data = EmailData(MandrillClient.USER_ACCOUNT_ACTIVATION_TEMPLATE, user.email, context)
+    MandrillClient().send_mail(email_data)
```

Comment:
These two lines are present in all of the six new functions added in this file. How about we create a new function containing these two lines and call it instead, thereby reducing code duplication.

Reply:
Done